tests/h/models/group_scope_test.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from sqlalchemy import inspect
+
+from h.models import GroupScope
+
+
+class TestGroupScope(object):
+    def test_save_and_retrieve_origin(self, db_session, factories):
```

Comment:
Hmm, some of these tests feel a bit like testing SQLAlchemy. I suppose there is still value because it checks for behaviour changes when eg. we upgrade SQLA or someone modifies the declarative configuration.

Reply:
Yeah. And to be fair - my first attempt at doing this class had totally unexpected and unwanted behaviours! Even though sqlalchemy is doing the actual work, I think it's worth having tests that I've used sqlalchemy as I think I've used it.

Reply:
(Another way to put this is that I'm not testing sqlalchemy, I'm testing the models that I've coded using sqlalchemy. Taking it to the extreme you could say that any tests of Python code are just "testing Python" :)